docs/release.md

```diff
@@ -334,7 +334,7 @@ cases, follow these steps below to ensure a proper patch release:
 - [ ] for each package (replace <kbd>carbon-components-react</kbd> with the
       package name):
   ```bash
-  npm dist tag add carbon-components-react@vX.Y.Z latest
+  npm dist-tag add carbon-components-react@vX.Y.Z latest
```

Comment:
I added this small change, hope that's okay. Something that's been bothering me for quite a while but have kept forgetting to put into a PR